feature-libs/asm/core/facade/asm.service.ts

```diff
@@ -53,6 +53,38 @@ export class AsmService {
     );
   }
 
+  /**
+   * Search for customers in a customer list
+   */
+  customerListCustomersSearch(options: CustomerSearchOptions): void {
```

Comment:
can we also move them to `AsmCustomerListService`?

Reply:
Not at the moment.  The models used are located in the `core` module while the facade is located in the `root` module.  Moving the methods would require a breaking change to move the models to `root`.